### DISCUSSION TITLE
Regrading field BT-02-notice and BT-03-notice while creating change notice.

### DISCUSSION THREAD

**Discussion by Nitin-789** [OP-TED/eForms-SDK#594] [Q&A] [answered]
Hi,
Currently we are working on sdk-1.5 version and for that we needed some confirmation regrading change notice. So for change notice we are setting up the value for BT-02-notice as "corr" and BT-03-notice as "change". So you'll be supporting that for version 1.5 right ?

Thanks and regrads,
Nitin

**Comment by ferrakaop** [comment, marked as answer]
This is correct: SDK 1.5 still requires "corr" and "change".  As from SDK 1.6, there are no rules that impose this - and we recommend to start using the original notice's form and notice types.  As from SDK 1.9, "corr" and "change" will be deprecated and can no longer be used.

**Reply by Nitin-789** [reply]
Thanks for the confirmation.
